packages/firestore/exp/src/api/database.ts

```diff
@@ -479,3 +489,41 @@ function verifyNotInitialized(firestore: FirebaseFirestore): void {
     );
   }
 }
+
+export function _loadBundle(
+  firestore: FirebaseFirestore,
+  bundleData: ArrayBuffer | ReadableStream<Uint8Array> | string
+): LoadBundleTask {
+  firestore._verifyNotTerminated();
+
+  const resultTask = new LoadBundleTask();
+
+  firestore._queue.enqueueAndForget(async () => {
+    const databaseId = (await firestore._getConfiguration()).databaseInfo
+      .databaseId;
+    const reader = createBundleReader(bundleData, newSerializer(databaseId));
+    const syncEngine = await getSyncEngine(firestore);
+
+    loadBundleSyncEngine(syncEngine, reader, resultTask);
+    return resultTask.catch((e: Error) => {
+      logWarn(LOG_TAG, `Loading bundle failed with ${e}`);
+    });
```

Comment:
This pattern means that the resulting user task never fails.  If you want to log an error, you should catch and rethrow here. A better way might be to log an error when the error occurs.
Note that you also should not return this task to the AsyncQueue, as it will cause it to fail. Instead, just return an empty Promise (which is the default behavior for async functions, so you can just remove `return`).

Reply:
I think this is fixed, although this particular piece of code no longer exists.